### DISCUSSION TITLE
Rule Type Change

### DISCUSSION THREAD

**Discussion by fast-turtle-9** [jertel/elastalert2#1461] [Q&A] [answered]
I have been using the change type rule and wanted to know if a certain behavior is possible. 
I am working with data in which a field can go from F to C and C to F. monitoring the change in the status. Is it possible to only alert when it goes from C to F and not when it goes from F to C or is it only possible to alert on both?
Tried looking for something in the documentation, but could not find anything. 
Thank you.

**Comment by jertel** [comment, marked as answer]
You can use an [enhancement](https://elastalert2.readthedocs.io/en/latest/recipes/adding_enhancements.html) to check the value and raise a DropMatchException if you don't want an alert to trigger.
